schema/types/isolator.go

```diff
@@ -1,6 +1,6 @@
 package types
 
 type Isolator struct {
-	Name ACName `json:"name"`
-	Val  string `json:"value"`
+	Name  string `json:"name"`
```

Comment:
er, I just realised you are doing conflicting things in this review: changing the isolator names _to_ `ACName`s (which is what it is in the explanatory part of the spec) but then changing this type back to a string - why?